charts/oidc-proxy/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.37
-appVersion: 0.0.36
+version: 0.0.38
+appVersion: 0.0.38
```

Comment:
Don't understand fully what's going on with this versioning, is `appVersion` leapfrogging over 0.0.37 on purpose?

Reply:
I think we typically keep them in sync.  I think it went out of sync with the manual change last week